src/Centreon/Domain/HostConfiguration/HostMacro.php

```diff
@@ -41,6 +43,7 @@ class HostMacro
 
     /**
      * @var bool Indicates whether this macro contains a password
+     * @EntityDescriptor(column="is_password", modifier="setPassword")
```

Comment:
should not be described in a yaml configuration file ?

Reply:
yes

Reply:
In fact, the yaml configuration file is used for the serializer.
This annotation will not be used by it but by the EntityCreator (Repository).

Reply:
We'll need to refactor EntityCreator cause we should avoid to use method annotation.
If I remember well, it's not compatible with ionCube

Reply:
> We'll need to refactor EntityCreator cause we should avoid to use method annotation.

Yes

> If I remember well, it's not compatible with ionCube

Only with the classes and methods annotations